### DISCUSSION TITLE
In user page links, what is the difference between regular and multi ?  در لینک صفحه کاربران تفاوت حالت معمولی و چندتایی چیه؟

### DISCUSSION THREAD

**Discussion by raminrez** [hiddify/Hiddify-Manager#154] [Q&A سوالات رایج]
آخرین نسخه 0.9.30 لینک‌های متفاوتی برای دسترسی به صفحه کابر تولید میکند
به طور مثال:
https://example.com/E5SJPUqfN89FMpjRvB7G/36e13d28-4e12-4339-879a-fce28da94c94/
https://example.com/E5SJPUqfN89FMpjRvB7G/36e13d28-4e12-4339-879a-fce28da94c94/multi

و تعداد کانکشن های داخل هر کدام متفاوت است. 
البته مدل جندتایی(multi) کانکشن کمتری داره. فرقشون چیه؟

**Comment by R30na** [comment]
مولتی همه کانکشن هایی که تو تنظیمات فعال کردی رو برای همه دامنه ها باهم میاره

**Reply by raminrez** [reply]
ولی توی کلش وقتی آپدیت میکنم چیزی اضافه نشده
و حتی نسبت به حالت نرمال تعداد کانکشن های کمتری داره
